sti/karaf/Dockerfile

```diff
@@ -38,10 +41,11 @@ COPY ./jolokia.properties /opt/jolokia/jolokia.properties
 COPY ./deploy-and-run.sh /opt/karaf/bin/
 RUN chmod a+x /opt/karaf/bin/deploy-and-run.sh
 
-# allow jboss user access to /opt/jboss
-RUN chown -R 1000 /opt/jboss
+# allow jboss user access to /opt/jboss and /opt/karaf
+RUN chown -R 1000 /opt/jboss \
```

Comment:
If these dirs need to be writeable by the user at runtime then you need to `chmod 777` them & `chmod 666` any non-executable files. The jboss user will not be used at runtime, rather an arbitrary uid will be used.